src/session/src/context.rs

```diff
@@ -129,6 +132,20 @@ impl QueryContext {
     pub fn set_current_user(&self, user: Option<UserInfoRef>) {
         let _ = self.current_user.swap(Arc::new(user));
     }
+
+    #[inline]
```

Comment:
We don't need `inline` anymore.
```suggestion
```

Reply:
All inline of this struct or just this function? Inline shouldn't have much effect, I just follow the code pattern

Reply:
We don't need to change the old codebase currently but appreciate it if you want. We don't add any `inline` in the new codebase.

Reply:
The `#[inline]` of this struct has been removed